### PR TITLE
Update list entity fields

### PR DESCRIPTION
**Pull request by JS-BC**

### CODE REVIEW

File: migrations/v2/V202410301210__v2.12.UpdateListFields.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+UPDATE [${flyway:defaultSchema}].[Entityfield]
+SET    DefaultInView = 1
+WHERE  ID IN ( 'C94217F0-6F36-EF11-86D4-6045BDEE16E6', -- List.ID
```
Comment (AN-BC) [inline migrations/v2/V202410301210__v2.12.UpdateListFields.sql:3]: @JS-BC  why are we including ID, ListID, RecordID values as DefaultInView, I could see the case for the List's Name and in the ListDetail, the ListDetail.List - which is a vritual field back to List.Name, Sequence maybe as well, but why all the others. DefaultInView should be the thigns that end users find valuable and most of these are UUIDs which are very hard for an end user to make sense of.

File: packages/Angular/Explorer/explorer-core/src/lib/app-view/application-view.component.ts [hunk 2]
```diff
@@ -372,7 +372,7 @@ export class ApplicationViewComponent extends BaseBrowserComponent implements On
                 this.sharedService.CreateSimpleNotification('There was an error linking to the shared view(s). Please try again later or notify a system administrator.', 'error', 3500);
             }
         }
-    }
+    }
```
Comment (AN-BC) [inline packages/Angular/Explorer/explorer-core/src/lib/app-view/application-view.component.ts:375]: @JS-BC  seems like a non-change unless I'm missing something here?

File: migrations/v2/V202410301210__v2.12.UpdateListFields.sql (hunk 1 above)
Comment (JS-BC) [reply migrations/v2/V202410301210__v2.12.UpdateListFields.sql:3]: @AN-BC The ID field is included in many entities, I was under the assumption that was standard. For ListID and RecordID, youre right. It is better to use the virtual fields

File: packages/Angular/Explorer/explorer-core/src/lib/app-view/application-view.component.ts (hunk 2 above)
Comment (JS-BC) [reply packages/Angular/Explorer/explorer-core/src/lib/app-view/application-view.component.ts:375]: Yes, I just made an empty change here for the purposes of running `npm run change` off of a single file

**Pull request merged**